### PR TITLE
Add landing message. #1

### PR DESCRIPTION
**Pull request by omar-alammari**

### CODE REVIEW

**Comment by omaralammari22** [comment]
leave a comment

File: index.html [hunk 1]
```diff
@@ -9,6 +9,7 @@
 
 <body>
     <p sty>welcome to team-dev</p>
+    <p>welcome to team-dev Hello</p>
```
Comment (dev-test22) [inline index.html]: cloud remove `Hello` welcome to team-dev string red

Comment (omar-alammari) [reply index.html]: sure, can do!

**Review by dev-test22 — CHANGES_REQUESTED** [review]
Please, change the code as per my comments on the review.

**Review by dev-test22 — APPROVED** [review]
Well done Omar!

**Pull request merged**